training/controllers/admin/admin_web.py

```diff
@@ -0,0 +1,17 @@
+from flask_admin import Admin
+from flask_admin.contrib.sqla import ModelView
+from training.models.users import User
+from training.app import db, app, bcrypt
+
+admin = Admin(app, name='Users admin', template_mode='bootstrap3')
+
+
+class MyModelView(ModelView):
```

Comment:
This class should have the name of the model, like `UserModelView` or something like that